worldwind-gt-geopkg/pom.xml

```diff
@@ -143,7 +143,13 @@
             <groupId>xmlunit</groupId>
             <artifactId>xmlunit</artifactId>
             <scope>test</scope>
-        </dependency>  
+        </dependency>
```

Comment:
I don't think this dependency is needed.  It was probably added by the IDE when were trying to resolve the imports. Please remove it to determine if it's really required.